tasks/setup.py

```diff
@@ -84,7 +98,22 @@ def setup(_: Context) -> None:
             break
         warn(f"{config.configParams.aws.publicKeyPath} is not a valid ssh key")
     
-    # Agent config
+    checkKeyPair = ask(f"Did you create your SSH key on AWS and want me to check it is loaded on your ssh agent when creating manual environments or running e2e tests [Y/N]? Default [{default_check_key_pair}]: ")
+    if len(checkKeyPair) > 0:
+        config.options.checkKeyPair = checkKeyPair.lower() == "y" or checkKeyPair.lower() == "yes"
+
+    if config.configParams.aws.teamTag is None:
+        config.configParams.aws.teamTag = ""
+    while True:
+        teamTag = ask(f"🔖 What is your team? It will tag all your resources by `TEAM:<team>`. Default [{config.configParams.aws.teamTag}]: ")
+        if len(teamTag) > 0:
```

Comment:
Why this test is needed as `if len(config.configParams.aws.teamTag) > 0:` performs the same test?

Reply:
This is meant to handle a setup with an existing config file. We load parameters from the config file and they become the default. If the user insert some input, I replace the value in the config map. If the user does not input anything, it means they want to keep the default, old value. Now we have two options: it is a fresh setup, user di not add anything, they need to pass something. Or we are in a second run of the setup, we keep the old value.